README.md

```diff
@@ -1563,6 +1562,13 @@ The example shows how to use custom metrics API in Python Backend. You can find
 the complete example instructions in
 [examples/custom_metrics](examples/custom_metrics/README.md).
 
+## PyTorch models
+
+The example shows how to use the
+[PyTorch platform handler](src/resources/platform_handlers/pytorch/README.md) to
+serve a PyTorch model. You can find the complete example instructions in
```

Comment:
```suggestion
serve a PyTorch model in Python backend. You can find the complete example instructions in
```

Reply:
Merged PyTorch docs into main Python docs.

Reply:
This section is integrated into the PyTorch platform section.